game-engine/src/main/scala/za/co/entelect/challenge/game/contracts/map/CarGameMap.scala

```diff
@@ -291,4 +289,9 @@ class CarGameMap(players: util.List[Player], mapGenerationSeed: Int, lanes: Int,
       carGamePlayer.finish();
     }
   }
+
+  private def findPositionToStartCountingColisionsFrom(oldPosition: BlockPosition): BlockPosition = {
+    val positionToStartCountingMud: BlockPosition = new BlockPosition(oldPosition.getLane(), oldPosition.getBlockNumber() + 1)
```

Comment:
@Arne27 - maybe stupid question, but does this work correctly when turning ?

Reply:
It should not affect the turning mechanism. I will check though, if you run into any issues please don’t hesitate to raise it